### PR TITLE
MB-1705: Release 2.0.0

### PR DESCRIPTION
**Pull request by rlepinski**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,33 @@
 UA Unity ChangeLog
 ==================
 
+Version 2.0.0 - March 23, 2016
+==============================
+
+New Features
+------------
+  - Updated interface to use c# Properties and Events
```
Comment (hcrowell) [inline CHANGELOG]: C# (capital C)?

**Comment by hcrowell** [comment]
:tangerine: after reviewing minor comment above ^

File: Assets/UrbanAirship/Platforms/CustomEvent.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ public void AddProperty(string name, ICollection<string> value)
 			this.propertyList.Add (new Property ("sa", name, value));
 		}
 
-		public string ToJson ()
+		internal string ToJson ()
```
Comment (marc-scig) [inline Assets/UrbanAirship/Platforms/CustomEvent.cs]: nice catch here

**Comment by marc-scig** [comment]
:bear:

**Comment by hcrowell** [comment]
:tangerine:

**Pull request merged**